src/pages/Error404.vue

```diff
@@ -10,7 +10,6 @@
       Sorry, nothing here...<strong>(404)</strong>
     </p>
     <q-btn
-      color="secondary"
       style="width:200px;"
```

Comment:
Do have it mobile-ready, `px` is not good. Instead, use `%` or `rem`.

Reply:
Good point. Let's do this in another PR.